src/client/src/services/credit/creditRfqs.ts

```diff
@@ -120,9 +135,9 @@ export const creditRfqsById$ = creditRfqUpdates$.pipe(
                   quotes: previousRfq.quotes.map((quote) => ({
                     ...quote,
                     state:
-                      quote.id === update.payload
-                        ? QuoteState.Accepted
-                        : QuoteState.Rejected,
+                      quote.id === update.payload.id
+                        ? ACCEPTED_QUOTE_STATE
```

Comment:
Looking through the new types `state` needs to be an object now `{type: typeof ACCEPTED_QUOTE_STATE, payload: number}`